users.js

```diff
@@ -120,29 +137,35 @@ module.exports = function (options) {
 
       args.mailingList = (args.mailingList) ? 1 : 0;
 
-      seneca.act({role: 'user', cmd: 'register'}, args, function (err, registerResponse) {
+      checkPassword(args, function (err, args) {
         if (err) return done(err);
-        if (!registerResponse.ok) {
-          return done(null, registerResponse);
+        if (typeof args.ok !== 'undefined' && !args.ok) {
```

Comment:
This is a custom function, to avoid any kind of "strange" scenario and avoid testing those, we could just enforce the value of the default return 
https://github.com/CoderDojo/cp-users-service/pull/189/files#diff-10aafeb1e619811e8b26a126d6ca2666R75